staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -88,24 +89,30 @@ var (
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
 	// TODO(audit): figure out a general way to let the client choose their preferred version
-	groupVersions = []schema.GroupVersion{auditv1beta1.SchemeGroupVersion}
-	registry      = registered.NewOrDie("")
+	registry = registered.NewOrDie("")
 )
 
 func init() {
-	registry.RegisterVersions(groupVersions)
-	if err := registry.EnableVersions(groupVersions...); err != nil {
-		panic(fmt.Sprintf("failed to enable version %v", groupVersions))
+	allGVs := []schema.GroupVersion{auditv1alpha1.SchemeGroupVersion, auditv1beta1.SchemeGroupVersion}
+	registry.RegisterVersions(allGVs)
+	numEnabled := 0
+	for _, gv := range allGVs {
+		if err := registry.EnableVersions(gv); err == nil {
+			numEnabled++
+		}
+	}
+	if numEnabled == 0 {
```

Comment:
if you restrict the enabled apis, at least one should be enabled, but maybe not all of them. But this is only important when we support some kind of negotiation.

Though now you ask I wonder: will we crash if alpha features are disabled?